### DISCUSSION TITLE
connection to oracledb via SOCKS proxy

### DISCUSSION THREAD

**Discussion by ciapecki** [oracle/python-oracledb#151] [General]
Hi,

Is it possible to leverage oracledb to connect to oracle database that is normally accessible via SOCKS proxy.
I have instructions for SQL Developer (thin jdbc client), but need to access it from within python.

In SQL Developer I needed to set:
oracle.net.socksProxyHost -> 127.0.0.1 (the tunnel is established via ssh)
oracle.net.socksProxyPort -> 1080
oracle.net.socksRemoteDNS -> true

I would be more than happy if someone would share how to use that to connect from within python with oracledb.

thanks

**Comment by ghuser** [comment]
Note: The above you need to set in the advanced section of each of connection properties section.

**Reply by anthony-tuininga** [reply]
This currently only works with JDBC. You can create an enhancement request if you would like to see this added to python-oracledb.
